addon/components/frost-toggle.js

```diff
@@ -53,45 +58,44 @@ export default Component.extend(SpreadMixin, PropTypeMixin, FrostEventsProxyMixi
   },
 
   getDefaultProps () {
+    const falseLabel = this.get('falseLabel')
+    const trueLabel = this.get('trueLabel')
+    const validTypes = ['string', 'number', 'boolean']
```

Comment:
Me either. The label should always be a string, and spec'd that way in propTypes in my opinion, but I didn't want to change functionality in this PR, just clean up the code :)